src/dvclive/live.py

```diff
@@ -380,6 +384,7 @@ def log_artifact(
         desc: Optional[str] = None,  # noqa: ARG002
         labels: Optional[List[str]] = None,  # noqa: ARG002
         meta: Optional[Dict[str, Any]] = None,  # noqa: ARG002
+        copy: bool = False,
```

Comment:
What's the path that's being copied in your scenario, and what would happen if you logged it with `copy=False`? Could we infer whether to copy?

Reply:
> What's the path that's being copied in your scenario

The path that you pass to the logger: `log_artifact('some_path')`

> and what would happen if you logged it with copy=False?

Same behavior as before. `.dvc` created nearby that `some_path` and in `dvc.yaml` we log it as `some_path`.

> Could we infer whether to copy?

not sure tbh, sounds a bit too much magic / complicated? Probably we would need a bit more data to decide.

Reply:
> > What's the path that's being copied in your scenario
> 
> The path that you pass to the logger: `log_artifact('some_path')`

Sorry, I meant for the example you are working on that prompted this PR. What is the exact path you want to log?

I'm wondering if it's inside/outside the repo, gitignored, etc.